### ISSUE TITLE
Ability to define Volume size on EC2?

### ISSUE THREAD

**Issue by scottalan** [puphpet/puphpet#1262]
Is there a way to provide the size of the volume you would like to attach to an EC2 instance? It seems the default is 8GB. Is there a `key: value` that you can put under:

```
provider:
   aws:
      volume_size: '30gb'
```

Not that exactly of course but really need to increase the size of the Volume when the instance is created.

Thanks

**Comment by scottalan** [comment]
Not that I'm aware of. When creating an instance through Amazon you
actually define the size of the volume. It defaults to 8gb. I had to create
a New volume on Amazon then migrate the 8gb volume to my new 30gb volume
manually.

**Comment by Boehemyth** [comment]
@jtreminio That is true for some instances, but EBS instances are not.  vagrant-aws provides a mechanism to specify the size of the EBS it should create.

Adding this inside the machine_id.vm.provider loop in Vagrantfile-aws allows me to set this in the config.yaml:
```
      aws.block_device_mapping      = [{'DeviceName' => "#{machine['block_device_mapping']}", 'Ebs.VolumeSize' => machine['block_device_size']}]
```
I'd be happy to submit a pull request if I knew where to add this in and what else might need to be done to make it work right.
